kaggle_environments/envs/football/football.json

```diff
@@ -1,87 +1,86 @@
 {
-    "name": "football",
```

Comment:
Spacing was wrong in this file, but the practical updates are basicallly the same as the other env json files.